client/src/pages/Profile.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Button, Typography, Tabs, Tab, Paper } from '@material-ui/core'
+import TabPanel from '../components/TabPanel'
+import ContestCard from '../components/ContestCard'
+
+const useStyles = makeStyles((theme) => ({
+    pageContainer: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center'
+    }, 
+    profileSection: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center'
+    }, 
+    contestSection: {
+        width: '80vw',
+        marginBottom: '4rem',
+        '& .MuiTab-fullWidth': {
+            fontSize: '1.2rem'
+        },
+        '& .MuiTab-textColorInherit.Mui-selected': {
+            fontWeight: 'bold'
+        }
+    },
+    avatar: {
+        width: '200px',
+        height: '200px',
+        borderRadius: '50%',
+        marginTop: '4rem'
+    },
+    userName: {
+        fontWeight: '600',
+        fontSize: '2rem',
+        margin: '1.5rem'
+    },
+    buttonDiv: {
+        textAlign: 'center',
+        margin: '0.5rem'
+    },
+    editButton: {
+        '& .MuiButton-label': {
+            fontWeight: 'bold'
+        }
+    }
+}))
+
+const tempData = [
+    {
+        image: 'tattoo-1.png',
+        noSketches: '24',
+        title: 'Lion tattoo concept in minimal style',
+        description: 'Looking for cool simplicity ideas of lion',
+        prizeAmount: '150'
+    },
+    {
+        image: 'tattoo-3.png',
+        noSketches: '24',
+        title: 'Cat tattoo concept in minimal style',
+        description: 'Looking for cool simplicity ideas of lion',
+        prizeAmount: '200'
+    },
+    {
+        image: 'tattoo-7.png',
+        noSketches: '24',
+        title: 'Lion tattoo concept in minimal style',
+        description: 'Looking for cool simplicity ideas of lion',
+        prizeAmount: '90'
+    },
+]
+
+export default function Profile() {
+    const classes = useStyles()
+    const [value, setValue] = useState(0)
```

Comment:
name this state something more meaningful such as `activeTab` as a recommendation.